qiskit_nature/problems/second_quantization/lattice/FermiHubbard.ipynb

```diff
@@ -0,0 +1,1270 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {}
+    }
+   ],
+   "metadata": {
+    "slideshow": {
+     "slide_type": "subslide"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### **adjacency matrix(hopping matrix) $t_{x, y}$**\n",
+    "The to_adjacency_matrix method returns its adjacency matrix.\n",
+    "By default, it returns a Hermitian matrix whose matrix elements are the weights.\n",
+    "Please note that the above graph has a weight $(4, 2, -1.0 + 2.0j)$, and the adjacency matrix $M$ has a matrix element $M_{2, 4} = -1.0 + 2.0j$."
```

Comment:
At least on Github the formatting of this cell is quite off (see screenshot below).
![screenshot_1634036615](https://user-images.githubusercontent.com/21973473/136944223-5c6ca5c2-33dc-4228-926c-5bdab61bb8ef.png)